react/Header/Header.js

```diff
@@ -32,9 +33,11 @@ export default function Header({
   const userClasses = classnames({
     [styles.user]: true,
     [styles.user_isReady]: authenticationStatus === UNAUTHENTICATED ||
-      (authenticationStatus === AUTHENTICATED && userName)
+      (authenticationStatus === AUTHENTICATED && userName || userEmail)
```

Comment:
Can you add parenthesis around `userName || userEmail`, otherwise it will match when even if the `authenticationStatus` is not `AUTHENTICATED`